app/models/update_existing_draft_edition.rb

```diff
@@ -79,4 +95,20 @@ def attributes
   def edition_attributes_from_payload
     payload.slice(*Edition::TOP_LEVEL_FIELDS)
```

Comment:
Actually, I wonder if we can just combined `TOP_LEVEL_FIELDS` with `ATTRIBUTES_TO_BE_ADDED` here to get a slice including those `publisher_` date fields?